selftests/functional/test_loader.py

```diff
@@ -215,6 +215,9 @@ def test_load_not_a_test(self):
     def test_load_not_a_test_not_exec(self):
         self._test('notatest.py', NOT_A_TEST, 'NOT_A_TEST')
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
I agree with disabling tests which depend on tight timeouts but this is not the case. The timeout here was set only to prevent OOM killer when Avocado misbehaves and for such cases increasing the timeout seems better solution.

As for other tests, sure, but please evaluate whether the test is really time-sensitive, or whether our initial timeouts were unnecessary strict. It's always better to keep the tests enabled (if possible) as we don't lose coverage of that particular area for `git bisect`.

Anyway we are not converging to a solution so let's talk in person on today's meeting and then we can either merge it (if you convince me that the other solution does not work) or let's increase the timeouts.